### ISSUE TITLE
[BUG] [Valheim] hostname needs surrounding " "

### ISSUE THREAD

**Issue by thegreatstorm** [GameServerManagers/LinuxGSM#3217]
Current one does not contain surrounding " " for `servername`
`startparameters="-name ${servername} -password ${serverpassword} -port ${port} -world ${gameworld} -public ${public}"`

Fix:
`startparameters="-name \"${servername}\" -password ${serverpassword} -port ${port} -world ${gameworld} -public ${public}"`

Current work around: -- add surrounding quotations.
servername="\"Valheim Server\""

**Comment by perru** [comment]
In what file please ?

**Comment by AnderssonChristian** [comment]
Assuming you are referring to `_default.cfg`, which you cannot edit, you can always copy the parameter into `vhserver.cfg` where you also set the server name, password, etc. So bascially:

`vhserver.cfg`:

```
servername="foo bar baz"
password="12345"
port="2456"
gameworld="${selfname}"
public="1"
startparameters="-name \"${servername}\" -password ${serverpassword} -port ${port} -world ${gameworld} -public ${public}"
```

**Comment by kingbeowulf** [comment]
I just set the dedicated Valheim server last night and have not noticed any issues, although my servername does not contain spaces. Is the the issue here?  XXXYYY versus XXX YYY?  I'm just inquiring for a bit more specificity on this issue report.

**Comment by liquidcowgithub** [comment]
+1 similar problem but with gameworld names when trying to restore a game world's files

added quotes for the world name in the startparameters propery in the vhserver.cfg

`startparameters="-name '${servername}' -password ${serverpassword} -port ${port} -world '${gameworld}' -public ${public}"`

**Comment by h3o66** [comment]
should be fixed in the next release with PR #3272
